README.md

```diff
@@ -5,11 +5,19 @@
   <em><b>Modelina</b></em>
 </h5>
 <p align="center">
-  <em>Modelina is the official AsyncAPI SDK used to generate different data models (i.e. <a href="#outputs">Java/TypeScript classes, Go Structs, etc</a>) for <a href="#inputs">AsyncAPI documents, among other supported inputs</a>.</em>
+  <em>Modelina is the official AsyncAPI SDK to generate data models (i.e. <a href="#outputs">Java/TypeScript classes, Go Structs, etc</a>) from <a href="#inputs">AsyncAPI documents, among other supported inputs</a>.</em>
 </p>
 
+
+[![blackbox pipeline status](https://img.shields.io/github/workflow/status/asyncapi/modelina/Blackbox%20testing%20(Stay%20Awhile%20and%20Listen)?label=blackbox%20testing)](https://github.com/asyncapi/modelina/actions/workflows/blackbox-testing.yml?query=branch%3Amaster++)
 [![Coverage Status](https://coveralls.io/repos/github/asyncapi/modelina/badge.svg?branch=master)](https://coveralls.io/github/asyncapi/modelina?branch=master)
-<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+[![Project Status: Active – The project has reached a stable, usable state and is being actively developed.](https://www.repostatus.org/badges/latest/active.svg)](https://www.repostatus.org/#active)
```

Comment:
I know that it's only a badge, but we have also note in the Readme:

![image](https://user-images.githubusercontent.com/20404945/155958954-10def3a2-c7a5-4bdc-866e-a2df881a414a.png)

and then we describe (as comment to the badge) `The project has reached a stable, usable state and is being actively developed.`

`stable` isn't good term in this place 😅

Reply:
That's honestly a good point, lets switch to `WIP` 🤔